modules/gke-cluster-autopilot/variables.tf

```diff
@@ -211,3 +211,9 @@ variable "vpc_config" {
   })
   nullable = false
 }
+
+variable "allow_net_admin" {
```

Comment:
Please move this into the `enable_features` variable.

Reply:
for the provider, one quick way is to update `default-versions.tf` then run `find . -name versions.tf -exec cp default-versions.tf "{}" \;`

Reply:
Is it recommended to update the beta and ga provider or just the ga in this instance. @ludoo

Reply:
we keep them both at the same version to avoid issues